### PR TITLE
Small arms REJOICE!!!!

### PR DESCRIPTION
**Pull request by fernandos13**
## About The Pull Request
this balances the small arms
smgs
machinepistols
pistols
revolvers

in many different and small improvements "revolver received a nerf to ROF and a BUFF to dmg" most of pitols got some level of buff tho not all.

## Why It's Good For The Game
will finally set a standard for small arms. tunning them around the overall balance.

makes most pistols comparable to one another, reduces the revolver from an unintended state to one comparable to the intended state, T19 to be closer to one another so it's less one being objectively worse, and more user preference
etc

buffs (TP-14,RT-3,TP-23,M1911,M9,TP-17,Highpower automag,88 Mod 4) to perform better one handed
also tweaks the T-19
T-19 now gets a small tweak to perform well one handed but being subpar overall than 2 handed guns

## Changelog
:cl:
balance: TP14 capacity increased.
balance: TP23, TP14, m1911 capacity increased, quicker fire rate and less recoil
balance: g22 accuracy increased, aimmode slowdown increased.
balance: TP17 damage increased
balance: Highpower automag recoil reduced, and less scatter when wielded
balance: 88mod 4 has quicker fire rate, less recoil, more scatter when not wielded
balance: most pistols have less scatter
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -14. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by TiviPlus** [comment]
freeze

**Comment by Pariah919** [comment]
I don't like the changes to revolvers, I think they're fine as is.
Pocket pistol uses its own bullet, don't obscfuate the code please.
Pistol ammo mag upgrade is fine

**Comment by TiviPlus** [comment]
Unatomized and broken changelog

File: code/modules/projectiles/guns/revolvers.dm [hunk 1]
```diff
@@ -342,7 +342,7 @@
 		/obj/item/attachable/lasersight,
 	)
 	attachable_offset = list("muzzle_x" = 33, "muzzle_y" = 19,"rail_x" = 13, "rail_y" = 23, "under_x" = 22, "under_y" = 14, "stock_x" = 22, "stock_y" = 19)
-	fire_delay = 0.3
+	fire_delay = 0.275
```
Comment (TiviPlus) [inline code/modules/projectiles/guns/revolvers.dm:345]: I will not accept this change

**Review by TiviPlus — CHANGES_REQUESTED** [review]

**Comment by SilencedMP5A5** [comment]
i like the changes to the other guns but leave revolver alone please

File: code/modules/projectiles/guns/revolvers.dm (hunk 1 above)
Comment (SplinterGP) [reply code/modules/projectiles/guns/revolvers.dm:345]: fix above please

Comment (fernandos13) [reply code/modules/projectiles/guns/revolvers.dm]: fixed

**Review by TiviPlus — APPROVED** [review]

File: code/modules/projectiles/guns/smgs.dm [hunk 2]
```diff
@@ -119,7 +119,7 @@
 	accuracy_mult = 1.15
 	accuracy_mult_unwielded = 0.8
 	scatter = 0
-	fire_delay = 0.15 SECONDS
+	fire_delay = 0.165 SECONDS
```
Comment (SplinterGP) [inline code/modules/projectiles/guns/smgs.dm:122]:
```suggestion
	fire_delay = 0.15 SECONDS
```

File: code/modules/projectiles/guns/smgs.dm [hunk 3]
```diff
@@ -72,8 +72,8 @@
 
 	attachable_offset = list("muzzle_x" = 27, "muzzle_y" = 17,"rail_x" = 9, "rail_y" = 20, "under_x" = 21, "under_y" = 12, "stock_x" = 24, "stock_y" = 10)
 
-	accuracy_mult = 1.5
-	accuracy_mult_unwielded = 0.85
+	accuracy_mult = 0.9
+	accuracy_mult_unwielded = 0.9
```
Comment (SplinterGP) [inline code/modules/projectiles/guns/smgs.dm:76]:
```suggestion
	accuracy_mult = 1.5
	accuracy_mult_unwielded = 0.85
```

**Review by SplinterGP — CHANGES_REQUESTED** [review]

**Review by SplinterGP — APPROVED** [review]

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-39). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**